src/app/repositories/repo-table/repo-table.component.ts

```diff
@@ -51,5 +40,87 @@ export class RepoTableComponent implements OnInit {
 			new TableHeaderItem({data: "Stars"}),
 			new TableHeaderItem({data: "Links"})
 		];
+
+		this.apollo.watchQuery({
+			query: gql`
+			  query REPO_QUERY {
+				# Let's use carbon as our organization
+				organization(login: "carbon-design-system") {
+				  # We'll grab all the repositories in one go. To load more resources
+				  # continuously, see the advanced topics.
+				  repositories(first: 75, orderBy: { field: UPDATED_AT, direction: DESC }) {
+					totalCount
+					nodes {
+					  url
+					  homepageUrl
+					  issues(filterBy: { states: OPEN }) {
+						totalCount
+					  }
+					  stargazers {
+						totalCount
+					  }
+					  releases(first: 1) {
+						totalCount
+						nodes {
+						  name
+						}
+					  }
+					  name
+					  updatedAt
+					  createdAt
+					  description
+					  id
+					}
+				  }
+				}
+			  }
+			`,
+		})
+		.valueChanges.subscribe((response: any) => {
+			if (response.error) {
+				const errorData = [];
+			 	errorData.push([
+					new TableItem({data: "error!" })
+				]);
+				this.model.data = errorData;
+			} else if (response.loading) {
+				this.skeleton = true;
+			} else {
+				// If we're here, we've got our data!
+				this.data = response.data.organization.repositories.nodes;
+				this.model.pageLength = 10;
+				this.model.totalDataLength = response.data.organization.repositories.totalCount;
+				this.selectPage(1);
+			}
+		});
+	}
+
+	selectPage(page) {
+		const offset = this.model.pageLength * (page - 1);
+		this.model.data = this.prepareData(this.data.slice(offset, offset + this.model.pageLength));
+		this.model.currentPage = page;
+	}
+
+	prepareData(data) {
+		this.skeleton = false;
+		const newData = [];
+
+		for (const datum of data) {
+			newData.push([
+				new TableItem({ data: datum.name, expandedData: datum.description }),
+				new TableItem({ data: new Date(datum.createdAt).toLocaleDateString() }),
+				new TableItem({ data: new Date(datum.updatedAt).toLocaleDateString() }),
+				new TableItem({ data: datum.issues.totalCount }),
+				new TableItem({ data: datum.stargazers.totalCount }),
+				new TableItem({
+					data: {
+						github: datum.url,
+						homepage: datum.homepageUrl
+					},
+					template: this.linkTemplate
+				})
+			]);
+		}
+		return this.model.data = newData;
```

Comment:
This again?

Based on the name of the function and the way it's used, you probably meant
```suggestion
		return newData;
```